sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClient.java

```diff
@@ -56,6 +59,59 @@
         this.encryptedBlobAsyncClient = encryptedBlobAsyncClient;
     }
 
+    /**
+     * Creates and opens an output stream to write data to the block blob. If the blob already exists on the service, it
```

Comment:
TODO : Change java doc

Reply:
Just clarifying that you are changing the Javadoc to say that it does not overwrite by default?

Reply:
This is straight copied from block blob. I thought I'd change it to say encrypted blob but the rest of the docs say block blob so I'll probably leave it that way